.yarn/patches/@metamask-network-controller-npm-18.0.1-c4d0cfaecd.patch

```diff
@@ -0,0 +1,55 @@
+diff --git a/dist/chunk-VNCJZRDU.js b/dist/chunk-VNCJZRDU.js
+index 78251fc9517438d90261ed53172d2a4133d3a5fd..d9fa7bc4b4fdc7d2cc9687cf9ad34c2e65cf809e 100644
+--- a/dist/chunk-VNCJZRDU.js
++++ b/dist/chunk-VNCJZRDU.js
+@@ -320,10 +320,14 @@ var NetworkController = class extends _basecontroller.BaseController {
+    * Populates the network clients and establishes the initial network based on
+    * the provider configuration in state.
+    */
+-  async initializeProvider() {
++  async initializeProvider(lookup) {
+     _chunkZ4BLTVTBjs.__privateMethod.call(void 0, this, _ensureAutoManagedNetworkClientRegistryPopulated, ensureAutoManagedNetworkClientRegistryPopulated_fn).call(this);
+     _chunkZ4BLTVTBjs.__privateMethod.call(void 0, this, _applyNetworkSelection, applyNetworkSelection_fn).call(this);
+-    await this.lookupNetwork();
++
++    lookup = lookup !== undefined ? lookup : true;
```

Comment:
Nit: Maybe simpler to drop this boolean. I don't anticipate us ever needing something like this; when we're ready to implement the alternative solution, we'd be dropping this patch.

The boolean param is confusing anyway; I almost suggested converting it to an options-bag style parameter so that we could see what it meant (e.g. `networkController.initializeProvider({ lookup: false })`), but easier to remove it.

Reply:
addressed in `36542422e37629e15cb61d2bbcdd5a0927b0e790`